### ISSUE TITLE
Big blocking error with Ubuntu 22.04

### ISSUE THREAD

**Issue by 4m1r7** [notthebee/ansible-easy-vpn#156]
Wolfgang said there might be an error in latest Ubuntu version, that can be ignored. But this one bring everything to halt, and I couldn't get rid of it whatever I tried.

```
TASK [system : Create locale en_US.UTF-8] *************************************************************************************
fatal: [localhost]: FAILED! =>
  msg: |-
    The task includes an option with an undefined variable. The error was: 'system_settings_locale' is undefined. 'system_settings_locale' is undefined

    The error appears to be in '/root/ansible-easy-vpn/roles/system/tasks/essential.yml': line 9, column 3, but may
    be elsewhere in the file depending on the exact syntax problem.

    The offending line appears to be:


    - name: Create locale {{ locale }}
      ^ here
    We could be wrong, but this one looks like it might be an issue with
    missing quotes. Always quote template expression brackets when they
    start a value. For instance:

        with_items:
          - {{ foo }}

    Should be written as:

        with_items:
          - "{{ foo }}"
```

**Environment:**
- Client OS: Linux
- VPS OS Ubuntu 22.04
- VPS Provider pq.hosting

**Comment by 4m1r7** [comment]
Thanks for the quick reply Wolfgang, I appreciate your time.

Unfortunately, I ran into another error, which I saw before trying to fix the issue myself (who I am kidding? with ChatGPT's help, of course!)

```
TASK [system : Create locale en_US.UTF-8] *************************************************************************************
fatal: [localhost]: FAILED! => changed=true
  cmd:
  - localedef
  - -i
  - en_US.UTF-8
  - -f
  - UTF-8
  - en_US.UTF-8
  delta: '0:00:00.401999'
  end: '2023-03-09 15:53:41.525301'
  msg: non-zero return code
  rc: 4
  start: '2023-03-09 15:53:41.123302'
  stderr: '[error] cannot open locale definition file `en_US.UTF-8'': No such file or directory'
  stderr_lines: <omitted>
  stdout: ''
  stdout_lines: <omitted>
```

**Comment by notthebee** [comment]
Try this:
```
locale-gen en_US.UTF-8
dpkg-reconfigure locales
```
And then re-run the bootstrap.sh script again

**Comment by swanserquack** [comment]
I've had the same error when setting up a server today today, running this fixed it

> Try this:
> 
> ```
> locale-gen en_US.UTF-8
> dpkg-reconfigure locales
> ```
> 
> And then re-run the bootstrap.sh script again

**Comment by notthebee** [comment]
Thanks for letting me know, the issue should be fixed by this PR: https://github.com/notthebee/ansible-easy-vpn/pull/151

**Comment by notthebee** [comment]
Hopefully fixed by 2840ed6a4b44b872b60983088a2e1cf8d787381d
